regression/regression_param.sh

```diff
@@ -24,13 +24,28 @@ case $machine in
            memnode=96
            numcore=40
     ;;
+<<<<<<< HEAD
```

Comment:
Looks like an unresolved conflict.  I think we want to retain lines 28 to 39  and git rid of lines 27, 40 to 48.